code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -534,7 +534,7 @@
 						RebuildHTML()
 
 					if("Money Account")
-						var/new_account = tgui_input_number(user, "What money account would you like to link to this card?", "Agent Card Account", 12345)
+						var/new_account = tgui_input_number(user, "What money account would you like to link to this card?", "Agent Card Account", 12345, max_value = 9999999)
```

Comment:
Why is it here 9 999 999? Nanobank app has 99 999

Reply:
99 999 is for the pin code, 9 999 999 is for changing the money account on a agent id.
Its the same bug at 2 places, may aswell fix it with this PR

Reply:
Oh, got it, thanks